media-library/overview/upload-files.md

```diff
@@ -45,6 +45,8 @@ If a new file is uploaded through the ImageKit media library, the uploader of th
 
 A restricted media library user cannot upload files at the root of the media library unless they are creating a new version of a file that exists at the root and is shared with at least "contribute" permission with them. Such a user needs at least "contribute" permission on the parent folder of any location to be able to create new folders.
 
+You can share a folder with a restricted user by providing "Can contribute" or "Can manage" access to it; this will allow them to start uploading their own assets.
+
 The file will not have any ownership info if it is not uploaded via the dashboard, e.g. via the API or an SDK. Note that when creating new file versions, the owner of the file does not change.
```

Comment:
Should this statement be up under Asset ownership section?

Reply:
done